go/tools/builders/go_path.go

```diff
@@ -112,6 +113,53 @@ func readManifest(path string) ([]manifestEntry, error) {
 	return entries, nil
 }
 
+func copySymlinkRecursively(src, dst string, writeFile func(src, dst string) error, createDir func(dst string) error) error {
+	target, err := os.Readlink(src)
+	if err != nil {
+		return err
+	}
+	if !filepath.IsAbs(target) {
+		target = filepath.Join(filepath.Dir(src), target)
+	}
+
+	return copyRecursively(target, dst, writeFile, createDir)
```

Comment:
Could this lead out us out of the sandbox? Would it work if we only copied the symlink as is without recurring on its target?

Reply:
> Could this lead out us out of the sandbox? 

I don't know enough about Bazel to answer this.

> Would it work if we only copied the symlink as is without recurring on its target?

I think that would only work for the `copyPath` function, but I don't think that would work for the ZIP archive created by `archivePath`, because the ZIP file cannot contain symlinks.